commands/log.go

```diff
@@ -105,10 +105,14 @@ func newLogCmd() *cobra.Command {
 			}
 		},
 	}
-	cmd.Flags().BoolVarP(&logForCurrentWeek, "week", "w", false, "show frames for entire week")
-	cmd.Flags().BoolVarP(&logForCurrentMonth, "month", "m", false, "show frames for entire month")
-	cmd.Flags().BoolVarP(&logForCurrentYear, "year", "y", false, "show frames for entire year")
-	cmd.Flags().BoolVarP(&logForYesterday, "yesterday", "d", false, "show frames for yesterday")
+	cmd.Flags().StringVarP(&logForWeek, "week", "w", "", "show frames for entire week")
+	cmd.Flags().Lookup("week").NoOptDefVal = time.Now().Format("2006-01-02")
```

Comment:
It looks like using `NoOptDefVal` breaks passing arguments separated by a space rather than an equal.

E.g `chrono report -m=2019-11` works, but `chrono report -m 2019-11` does not:


There is an open issue regarding this: https://github.com/spf13/cobra/issues/866

But it doesn't seem there is much movement on it.

Until it does get fixed upstream, perhaps we handle injecting default values in Chrono itself?

If we leave it as is, it'll mean inconsistency in the CLI's interface as everywhere else you could use spaces (which is what I personally use as well).

Reply:
Ahh alright. I've subscribed to that issue to follow up on it, and I'll add code to inject the default values in Chrono. Thanks for catching it!

Reply:
This is an issue in `pflag` rather than in `cobra`: https://github.com/spf13/pflag/issues/134, and it doesn't seem like there's any plans to address it.

I'm not yet too sure how to handle this in Chrono itself, as it doesn't seem like `pflag` would support the syntax we want, see [here](https://github.com/spf13/pflag#setting-no-option-default-values-for-flags) and [here](https://github.com/spf13/pflag#command-line-flag-syntax).

Reply:
That's unfortunate -  I'm going to go ahead and merge this commit but I've opened an issue to track the inconsistent flag usage. Perhaps a solution can be found in the future.